packages/cli/src/test/prompts/prompt.test.js

```diff
@@ -273,5 +274,37 @@ describe('prompt', function() {
         });
       });
     });
+
+    describe('#proxyInfo', function() {
+      beforeEach('initialize projectFile', function() {
+        this.projectFile = new ProjectFile('mocks/mock-stdlib-2/zos.json');
+      });
+
+      this.afterEach('restore stubs', function() {
```

Comment:
Did you add `this.` for any reason in particular?

```suggestion
      afterEach('restore stubs', function() {
```